### ISSUE TITLE
Add "Online" Filter

### ISSUE THREAD

**Issue by travis-adams** [travis-adams/neighborhood-pulse#42]
The "online" checkbox is present in the frontend right now:
![image](https://user-images.githubusercontent.com/39063700/95003664-3dd95100-05af-11eb-8fa7-f00288cf0e6b.png)
but it does nothing.

This ticket is for adding the "/online" parameter to the HTTP request sent when the box is checked. The fix will involve either changing the "fetchFilteredEvents" method in EventService.tsx, or creating a new method maybe called "fetchOnlineEvents" 

I'm thinking if "online" is checked, then the map grays out and the event grid expands a bit. I'm not sure whether the event details panel should also expand in size. Currently the event grid is 40%, map is 60%, and the detail panel expands from 0 to 40% when opened, covering 2/3 of the map. Maybe "online" would make the grid, map, and details all 50% width so that opening the details panel completely covers the map

Basically, I want the user's focus to transfer from the map to the event list. Do that however you want
